backend/devices/bitbox02/device.go

```diff
@@ -145,3 +149,14 @@ func (device *Device) Reset() error {
 	device.init()
 	return nil
 }
+
+func (device *Device) CreateBackup() error {
```

Comment:
Wouldn't you also want this for `EraseBackup`?

Reply:
> `EraseBackup`

where is this magical function :joy: 

Kidding aside, bb02 does not support deleting backups. We deemed it most important to keep all backups so there are no accidentally deleted backups that turned out to have funds.

Reply:
hehe. I just found it here with a grep: https://github.com/digitalbitbox/bitbox-wallet-app/blob/b6030d7203fbd6df2d8b2278d02666612186757a/backend/devices/bitbox/device.go#L936

But I didn't realize this is probably bitbox01?

Reply:
So, basically all devices are in bitbox02-api-go except bb01. Maybe some cleanup in the future?

Reply:
not sure what you mean? bitbox02-api-go is bitbox02 only (firmware and bootloader).

`backend/devices/bitbox` is bitbox01 (implementing the `Device` interface)
`backend/devices/bitbox02*` is bitbox02 (using bitbox02-api-go to implement the `Device` interface`)

Reply:
well, there's also bitboxbase. it's probably obvious to you that it's also bb02 but I kind of forgot.

no but my point was maybe move bb01 stuff in a separate repo? if it makes sense.

Reply:
Got no reason to move it to a different repo, it is only used here. Would be just needless overhead.

bitbox02-api-go was moved to a repo so it can be used in bitbox02-api-js.

Reply:
> well, there's also bitboxbase. it's probably obvious to you that it's also bb02 but I kind of forgot.

Right, didn't think of it. Maybe more accurately, bitbox02-api-go is for all firmware API coming out of bitbox02-api-firmware ;)